### PR TITLE
Remove the hard-coded ARG in aos_sal Dockerfile.

### PR DESCRIPTION
**Pull request by teweitsai**

### CODE REVIEW

File: aos_sal/Dockerfile [hunk 1]
```diff
@@ -1,20 +1,22 @@
 # Dockerfile to execute a base machine with all set to test active optics
 # system software
 
-FROM lsstts/develop-env:c0010
+ARG IMAGE_TAG
```
Comment (pkubanek) [inline aos_sal/Dockerfile:4]: Can you please provide default value? develop-env:latest?

**Review by pkubanek — APPROVED** [review]
Looks good, but consider providing default. If you don't provide default, please document in README.md

File: aos_sal/Dockerfile (hunk 1 above)
Comment (teweitsai) [reply aos_sal/Dockerfile:4]: The problem is that Tiago will never follow this (`latest`) in `develop-env`. You can see he always ask us to use `c0015` or `b045` or something like this~ I guess you should try to persuade him first. Actually, I also felt it is annoying that I never understand the meaning of `c0xx` or `b0xx`.

**Comment by teweitsai** [comment]
I documented the `IMAGE_TAG` is needed in the `README.md` already. Thanks!

**Pull request merged**